packages/flutter/lib/src/rendering/block.dart

```diff
@@ -187,6 +189,8 @@ class RenderBlock extends RenderBox
       case Axis.vertical:
         return _getIntrinsicCrossAxis((RenderBox child) => child.getMinIntrinsicWidth(height));
     }
+    assert(mainAxis != null);
```

Comment:
@Hixie and I chatted about these cases and his suggestion (I think) was to assert not `null`.  Happy to do whatever you guys agree on though! 👍